packages/react-ai/src/components/AIConversation/context/MessagesContext.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { ConversationMessage } from '../types';
+import { ConversationMessage } from '../../../types';
 
 type MessagesContextProps = ConversationMessage[] | undefined;
```

Comment:
Will the customer need this type?

Reply:
I'm not too sure yet. Could be useful in the case where they used the `AIConversation` component without using our hooks.

Reply:
Can we add a follow up task to audit our exported types before release?

Reply:
will do